locales/fr.json

```diff
@@ -137,5 +137,9 @@
   "No recording": "Aucun enregistrement",
   "questions-2.title": "questions-2.title",
   "questions-2.intro": "questions-2.intro",
-  "footer.link1.desc": "footer.link1.desc"
+  "footer.link1.desc": "footer.link1.desc",
+  "asdf": "asdf",
+  "form.translation": "English translation of: \"{{ text }}\"",
+  "translations-1.title": "translations-1.title",
+  "translations-1.intro": "translations-1.intro"
```

Comment:
Is the French for "asdf" really "asdf"?
`form.translation` and `translations-1.*` could use some translating too.